### PR TITLE
Clear README file

### PR DESCRIPTION
**Pull request by marcinciapa**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1 +1 @@
-This is an empty Readme file
\ No newline at end of file
+test
```
Comment (marcinciapa) [inline README.MD:1]:
```suggestion
tests
```

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

File: README.MD [hunk 2]
```diff
@@ -1 +1 @@
-This is an empty Readme file
\ No newline at end of file
+tests
```
Comment (marcinciapa) [inline README.MD:1]:
```suggestion
testss
```

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

File: README.MD [hunk 3]
```diff
@@ -1 +1 @@
-This is an empty Readme file
\ No newline at end of file
+testss
```
Comment (marcinciapa) [inline README.MD:1]:
```suggestion
testsss
```

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by mciapa** [comment]
ready for release

**Comment by mciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Comment by marcinciapa** [comment]
ready for release

**Pull request merged**